cime_config/tests.py

```diff
@@ -305,6 +305,8 @@
             "SMS_D_Ld1.ne30pg2_r05_EC30to60E2r2.WCYCL1850",
             "SMS_Ln5.ne30pg2_ne30pg2.F2010-SCREAM-LR-DYAMOND2",
             "ERS_Ld3.ne30pg2_r05_EC30to60E2r2.WCYCL1850.allactive-nlmaps",
+            "SMS_D_Ld1.ne30pg2_r05_IcoswISC30E3r5.CRYO1850",
```

Comment:
@darincomeau, I think we should move this one to production and leave DISMF here.  Do you want me to do that?

Reply:
Thanks!